### DISCUSSION TITLE
GD-25B256EYIG GigaDevice

### DISCUSSION THREAD

**Discussion by Pyro1992** [nofeletru/UsbAsp-flash#114] [General]
hey, got an Gigabyte G5 KF Laptop with an "GigaDevice, AJ2247, 25B256EYIG, UHJ814" BIOS chip. But i cannot find it in the List. Is it not supported yet?
![Screenshot_20240224_215928_Gallery](https://github.com/nofeletru/UsbAsp-flash/assets/18252895/d0d3e1b8-1fb6-4eb7-b65a-00aa49b0e439)

**Comment by arieslaptop** [comment]
chip id ?
![image](https://github.com/nofeletru/UsbAsp-flash/assets/88767405/724dfb53-4b09-4f38-be7d-18f9706d0524)

**Comment by Pyro1992** [comment]
How to read this id out? Ibtryd several profiles bot no ones detects / readout the chip. Everytime.not recognized / connected

**Comment by arieslaptop** [comment]
Click here , and which id detect in bottom box check it . send me screenshot 

![image](https://github.com/nofeletru/UsbAsp-flash/assets/88767405/757d267b-740c-4031-91e1-b5a687d7e9a7)

**Reply by Pyro1992** [reply]
![13-03-2024 17_38_03-NVIDIA GeForce Overlay DT](https://github.com/nofeletru/UsbAsp-flash/assets/18252895/80c316bb-d3e6-4a22-a9ee-54980f3b10ff)

i cannot read it out. Everytime 

Current programmer: CH341
ID(9F): FFFFFF(Unknown)
ID(90): FFFF(Unknown)
ID(AB): FF(Unknown)
ID(15): FFFF(Unknown)

It´s definitly seats correct. Tryd 7 times. Driver is also installed correctly and the Programmer is pinned like this ones (in the 25xx slot also pin 1 red is connected to 1 on bios)
[https://www.hardwareluxx.de/community/attachments/7_3-jpg.721969/](url)

**Reply by arieslaptop** [reply]
This read ID information means no pins or any one pin are not connected to the clip. This clip connected on sop8 for wson8 it mostly fails to contact, 
remove bios from the motherboard and read it from the programmer

**Comment by rgekhman** [comment]
Sorry, late reply. 
It won't grab onto the pins if clamped on pcb. 
Remove the chip from pcb then used the NeoProgrammer with a clamp.

[NeoProgrammer V2.2.0.10.zip](https://github.com/user-attachments/files/28279190/NeoProgrammer.V2.2.0.10.zip)

<img width="651" height="353" alt="image" src="https://github.com/user-attachments/assets/26ddddf9-174d-410a-9e29-e7a26457fbc6" />
